client/src/apps/code-collabo/styles/app.main.js

```diff
@@ -68,6 +86,19 @@ export default css.global`
             border-right: 1px solid var(--color-neutral-200);
         }
 
+        .app__logo {
+            margin-bottom: 82px;
+        }
+
+        .app__menubar__nav__link {
+            padding: 15px 0px;
+            gap: 30px;
+        }
+
+        .app__menubar__nav__items {
+            gap: 20px
```

Comment:
**Desktop:**

Please **add** width here (instead of padding for this `.app__menubar__nav__items` container item. As I have said we don't want to use padding on the left and right of container items).

Explanation:
On the figma design the sidebar width for **desktop** is 360, and then the width of the button(s) is 300. The space at the side is 60 (i.e. 30 on the left and 30 on the right).

In the code, you can use the idea to get the percentage of the `.app__menubar__nav__items` container for desktop - use that as width (you can just use the CSS calc() function for this).
The use the auto margin on the `.app__menubar__nav__items` container to centralize it - that way you don't have to bother about what the sides will be as they are just set dynamically. Saying this here also because that gap spacing is incorrect for desktop based on "Explanation" above.

Reply:
@Ifycode Done